src/main/java/org/opentripplanner/common/MinMap.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.common;
+
+import java.util.HashMap;
+
+/**
+ * A HashMap that has been extended to track the greatest or smallest value for each key.
+ * Note that this does not change the meaning of the 'put' method. It adds two new methods that add the min/max
+ * behavior. This class used to be inside SimpleIsochrone.
+ */
+public class MinMap<K, V extends Comparable<V>> extends HashMap<K, V> {
```

Comment:
I think this should be named MinMaxMap

Reply:
I agree, I think this is just an effect of accumulated features over time, without changing class names.